Dockerfile

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 FROM ubuntu:xenial
 
-RUN apt-get update && apt-get install -y python3-pip curl
-RUN pip3 install tensorboard
+RUN apt-get update && apt-get install -y python-pip curl
```

Comment:
Python 3 should be used wherever possible. Python 2 is in maintenance only mode.

Reply:
Fair enough :)